packages-ts/gauntlet-terra-contracts/src/commands/contracts/ocr2/inspection/inspect.ts

```diff
@@ -0,0 +1,165 @@
+import { getOffchainConfigInput, OffchainConfig } from '../setConfig'
+
+type Expected = {
+  description: string
+  decimals: string | number
+  minAnswer: string | number
+  maxAnswer: string | number
+  transmitters: string[]
+  billingAccessController: string
+  requesterAccessController: string
+  link: string
+  billing: {
+    observationPaymentGjuels: string
+    recommendedGasPrice: string
+    transmissionPaymentGjuels: string
+  }
+  offchainConfig: OffchainConfig
+}
+
+const makeInput = async (flags: any, args: string[]): Promise<InspectionInput<any, Expected>> => {
+  if (flags.input) return flags.input as InspectionInput<any, Expected>
+  const rdd = getRDD(flags.rdd)
+  const contract = args[0]
+  const info = rdd.contracts[contract]
+  const aggregatorOperators: string[] = info.oracles.map((o) => o.operator)
+  const transmitters = aggregatorOperators.map((operator) => rdd.operators[operator].ocrNodeAddress[0])
+  const billingAccessController = flags.billingAccessController || process.env.BILLING_ACCESS_CONTROLLER
+  const requesterAccessController = flags.requesterAccessController || process.env.REQUESTER_ACCESS_CONTROLLER
+  const link = flags.link || process.env.LINK
+  const offchainConfig = getOffchainConfigInput(rdd, contract)
+  return {
+    expected: {
+      description: info.name,
+      decimals: info.decimals,
+      minAnswer: info.minSubmissionValue,
+      maxAnswer: info.maxSubmissionValue,
+      transmitters,
+      billingAccessController,
+      requesterAccessController,
+      link,
+      offchainConfig,
+      billing: {
+        observationPaymentGjuels: info.billing.observationPaymentGjuels,
+        recommendedGasPrice: info.billing.recommendedGasPrice,
+        transmissionPaymentGjuels: info.billing.transmissionPaymentGjuels,
+      },
+    },
+  }
+}
+
+const makeOnchainData = (instructionsData: any[]): Expected => {
+  const latestConfigDetails = instructionsData[0]
+  // TODO: Offchain config is not stored onchain, only the digested config. Gauntlet could calculate the digested with RDD values and compare it
+  // const offchainConfig = deserializeConfig(latestConfigDetails.config_digest)
+  const description = instructionsData[1]
+  const transmitters = instructionsData[2]
+  const decimals = instructionsData[3]
+  const billing = instructionsData[4]
+  const billingAC = instructionsData[5]
+  const requesterAC = instructionsData[6]
+  const link = instructionsData[7]
+
+  return {
+    description,
+    decimals,
+    minAnswer: 'INFO NOT AVAILABLE IN CONTRACT',
+    maxAnswer: 'INFO NOT AVAILABLE IN CONTRACT',
+    transmitters: transmitters.addresses,
+    billingAccessController: billingAC,
+    requesterAccessController: requesterAC,
+    link,
+    offchainConfig: {} as OffchainConfig,
+    billing: {
+      observationPaymentGjuels: billing.observation_payment,
+      transmissionPaymentGjuels: 'INFO NOT AVAILABLE IN CONTRACT',
```

Comment:
This should be available on-chain as part of billing config right? There are also some new configs in Blaz's latest PR (gas reimbursements).